cirq/protocols/has_stabilizer_effect_protocol.py

```diff
@@ -59,3 +64,17 @@ def _strat_has_stabilizer_effect_from_gate(val: Any) -> Optional[bool]:
     if hasattr(val, 'gate'):
         return _strat_has_stabilizer_effect_from_has_stabilizer_effect(val.gate)
     return None
+
+
+def _strat_has_stabilizer_effect_from_unitary(val: Any) -> Optional[bool]:
+    """Attempts to infer whether val has stabilizer effect from its unitary.
+
+    Returns whether unitary of `val` normalizes the Pauli group. Works only for
+    2x2 unitaries.
+    """
+    if not protocols.has_unitary(val):
+        return None
+    unitary = protocols.unitary(val)
+    if unitary.shape == (2, 2):
+        return not SingleQubitCliffordGate.from_unitary(unitary) is None
```

Comment:
nit: use `is not None` for better readability:

```python
return SingleQubitCliffordGate.from_unitary(unitary) is not None
```

Reply:
Done.